File: C++ Programming/problems on numberical/problem22.cpp
/*

22. Write a program which accept area in square feet and convert it into square meter.
	(1 square feet = 0.0929 Square meter) 
	
	Author : Milind Nalawade
	
	Date:26/06/2021
	
	Language : C++ programming
	
*/	
using namespace std;
#include<iostream>

//Logic
float square_meter(float SF)
{
	return SF*0.0929;
}

// Entry point function

int main()
{
	float SF=0.0,ret=0.0;
	printf("Enter area in square feet : ");
	scanf("%f",&SF);
	
	ret=square_meter(SF);
	printf("area in square meter is : %f",ret);
	return 0;
}

File: Python Programming/problems on numbericals/problem22.py
'''

22. Write a program which accept area in square feet and convert it into square meter.
	(1 square feet = 0.0929 Square meter) 

	Author : Milind Nalawade
	
	Date:26/06/2021
	
	Language : Python 3.9.0
	
'''
#======
#Logic
#======

def  Square_meter(SF):
	return SF*0.0929;

#=============
#Entry point function
#=============

def main():
	F=float(input("Enter area in square feet : "))
	
	ret= Square_meter(F)
	print("rea in square meter is : ",ret)
	
#======
#starter
#======
if __name__=="__main__":
	main()